docs/containers-policy.json.5.md

```diff
@@ -10,8 +10,7 @@ containers-policy.json - syntax for the signature verification policy file
 Signature verification policy files are used to specify policy, e.g. trusted keys,
 applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
 
-The default policy is stored (unless overridden at compile-time) at `/etc/containers/policy.json`;
-applications performing verification may allow using a different policy instead.
+The default policy is stored (unless overridden at compile-time) at `/etc/containers/policy.json`;  applications performing verification may allow using a different policy instead. Rootless applications may also read the default policy from `$HOME/.config/containers/policy.json` if it exists.
```

Comment:
I think I'd change "may also read" -> "will read"

Reply:
“rootless” is a Podman jargon, and anyway this does not accurately describe what the code does.

Reply:
how about mention `$HOME/.config/containers/policy.json` if it exists. Otherwise, will use the `/etc/containers/policy.json`?

Reply:
Yes from containers image point of view, we should read the file whether we are root or rootless.
IE if we are root we should look for /root/.config/contianers/policy.json
So we don't really need to differentiate.

Reply:
PTAL , I updated the containers-policy.json.5.md